nicegui/native/native_mode.py

```diff
@@ -21,14 +21,32 @@
         # webview depends on bottle which uses the deprecated CGI function (https://github.com/bottlepy/bottle/issues/1403)
         warnings.filterwarnings('ignore', category=DeprecationWarning)
         import webview
+        from webview.dom import DOMEventHandler
     optional_features.register('webview')
 except ModuleNotFoundError:
     pass
 
 
+def on_drop(e: dict[str, Any], drop_queue: mp.Queue):
+    files = e['dataTransfer']['files']
+    if len(files) == 0:
+        return
+
+    for file in files:
+        drop_queue.put(file.get('pywebviewFullPath'))
```

Comment:
```suggestion
    for file in e['dataTransfer']['files']:
        drop_queue.put(file.get('pywebviewFullPath'))
```

The early exit is not needed. If there are no files, the for-loop doesn't do anything.